test/e2e/tests/multichain/permission-page.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { DAPP_HOST_ADDRESS } from '../../constants';
+import { withFixtures } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import PermissionListPage from '../../page-objects/pages/permission/permission-list-page';
+import SitePermissionPage from '../../page-objects/pages/permission/site-permission-page';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+
+describe('Permissions Page', function () {
+  it('should redirect users to connections page when users click on connected permission', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver }) => {
+        await loginWithBalanceValidation(driver);
```

Comment:
Given there's no transaction involved, I think we can login without waiting for the balance to be loaded. This can speed up a bit the test

```suggestion
        await loginWithoutBalanceValidation(driver);
```

Reply:
Done